flask_resty/view.py

```diff
@@ -144,6 +145,47 @@ def get_data_id(self, data):
 
         return tuple(data[id_field] for id_field in self.id_fields)
 
+    def get_query_data(self, **kwargs):
+        args = flask.request.args
+        data_raw = {}
+
+        for field_name, field in self.query_schema.fields.items():
+            if field_name in args:
+                args_key = field_name
+            elif field.load_from and field.load_from in args:
+                args_key = field.load_from
+            else:
+                continue
+
+            value = args.getlist(args_key)
+            if not self.is_list_field(field) and len(value) == 1:
```

Comment:
I would probably inline it instead of having an `is_list_field` method on the view

Reply:
it's to allow overriding, just in case

Reply:
why do you think anyone would need to override this tho?

Reply:
like if they have a different list field

Reply:
wouldn't that be a subclass of list most probably? if you feel that can happen anyway, 👍

Reply:
or if it's not marshmallow... you can actually override all the marshmallow hooks here i think